OpenRA.Mods.Common/Widgets/DropDownButtonWidget.cs

```diff
@@ -19,8 +20,10 @@ namespace OpenRA.Mods.Common.Widgets
 {
 	public class DropDownButtonWidget : ButtonWidget
 	{
-		public readonly string SeparatorCollection = "dropdown";
-		public readonly string SeparatorImage = "separator";
+		public readonly string Arrows = "arrows";
+		public readonly string ArrowImage = "down";
+		public readonly string Separators = "separators";
```

Comment:
IMO it would probably be tidier to instead have:
```
public readonly string Decorations = "dropdown-decorations";
public readonly string DecorationMarker = "marker";
public readonly string DecorationSeparator = "separator";
```
and
```
dropdown-decorations:
	Inherits: ^Chrome
	Regions:
		marker: 836, 17, 16, 16
		marker-pressed: 853, 17, 16, 16
		marker-disabled: 853, 17, 16, 16
```

rather than mixing concepts between dropdowns and scrollpanels.

Reply:
I guess we could also have a single `button-decorations` image that covered both dropdowns and scrollpanels.

Reply:
This setup won't work when the marker and the separator come from different sprites. This is the case in every mod except TD and the reason why I made two separate properties for image collections.

So, perhaps go with something like:
```csharp
public readonly string Marker = "dropdown-marker";
public readonly string MarkerImage = "marker";
public readonly string Separator = "dropdown-separator";
public readonly string SeparatorImage = "separator";
```

Reply:
Hmm, i see. It might be simpler to just add the separator to glyphs.svg?

Reply:
I don't have a strong objection against that. We'd have to duplicate several small parts of the dialog graphics and that's somewhat fiddly and more difficult to maintain. But we'd get nicer semantics.

The separator is such an odd piece... I think it should be part of the marker graphic. But it's not because the marker is assembled from a separator and an arrow.

Reply:
I wouldn't object to combining them.

Reply:
I think that moving the separator to the glyph sprite (on its own or combined with the marker) is more trouble than it's worth :thinking: The separators are using the dialog art and moving them away would just incur maintenance cost. And furthermore they are an optional bit and mods can choose to ignore them and just use a marker image.

I suggest we leave it like this:
```csharp
public readonly string Decorations = "dropdown-decorations";
public readonly string DecorationMarker = "marker";
public readonly string Separators = "dropdown-separators";
public readonly string SeparatorImage = "separator";
```
Of course if you insist @pchote I will move them to the glyph sprite.

Reply:
Ok, if we can unify the other cases then i'm fine with leaving this one split.